src/interfaces/staking/definitions.ts

```diff
@@ -9,6 +9,7 @@ export default {
                 Controller: 'Null',
                 Account: 'AccountId'
             }
-        }
+        },
+        VecDeque: 'Vec<BalanceOf<any>>',
```

Comment:
Added this type definition, rest of file changes are autogenerated.
Used any here as - any is certainly generic in that it will cause the function to accept any and all types for the type.
https://github.com/cennznet/cennznet/blob/48b965cff4463ff0be81e14b826d24046615f2db/crml/staking/src/rewards/mod.rs#L78

Reply:
Is it possible to have this as `VecDeque: 'Vec<any>'`? VecDeque shouldn't really be tied to balance at all.
If we can't solve this here in API side then we can make a type alias for it on node side e.g. 'RewardTxHistory'

Reply:
Vec<any> worked.. I have updated it here and used it in API, it builds good and on connect I don't get any warning

Reply:
perfect 👍